### DISCUSSION TITLE
Wiget is still downloading V4.4.170

### DISCUSSION THREAD

**Discussion by trexpark** [Open-Shell/Open-Shell-Menu#1635] [General]
Trying to install the open-Shell-menu via Winget however the installer is pointing to version 4.4.170. The latest version is 4.4.190.
Not sure if this is a Winget issue or if Winget repo needs to be updated

**Found Open-Shell-Menu [Open-Shell.Open-Shell-Menu]
Version: 4.4.170
Publisher: Open-Shell
Moniker: openshellmenu
Description: Classic style Start Menu for Windows 7, 8, 8.1, 10
Homepage: https://github.com/Open-Shell/Open-Shell-Menu
License: MIT
License Url: https://github.com/Open-Shell/Open-Shell-Menu/blob/master/LICENSE
Release Notes Url: https://github.com/Open-Shell/Open-Shell-Menu/releases/tag/v4.4.170
Tags:
  customize
  start-menu
Installer:
  Installer Type: exe
  Installer Url: https://github.com/Open-Shell/Open-Shell-Menu/releases/download/v4.4.170/OpenShellSetup_4_4_170.exe
  Installer SHA256: 0417041cffed3478f13a840e45ee304ccf8d1f9ca38e4126bc315161ac9f1669
  Release Date: 2022-05-27**

**Comment by ge0rdi** [comment]
Hello, we are not submitting any packages to 3rd party systems.

Feel free to submit WinGet PR here:
https://github.com/microsoft/winget-pkgs
